models/requests.ts

```diff
@@ -76,41 +78,90 @@ export const updateRequest = async (id: string, body: any, lastModifiedBy: strin
   }
 };
 
-export const getRequests = async (query: RequestQuery) => {
-  const { id, type, requestedBy, state } = query;
+export const getRequests = async (query: any) => {
+  let { id, type, requestedBy, state, prev, next, page, size = SIZE } = query;
+  size = parseInt(size);
+  page = parseInt(page);
```

Comment:
pagination logic in the `getRequests` function appears complex and could be challenging to understand at first.

You can consider simplifying the logic and breaking it down into smaller, more manageable functions to improve readability and maintainability.

Also please ensure comprehensive handling of edge cases, such as scenarios where there are no more previous or next pages available.

Reply:
Can you explain a little how we can make this easier?